### DISCUSSION TITLE
Deployed fully but the URL Manager doesn't load in Tiny Blazor Admin

### DISCUSSION THREAD

**Discussion by kylerosenthal** [microsoft/AzUrlShortener#439] [Q&A]
Using the new deployment - deployed AzUrlShortener and TinyBlazorAdmin. 

I have added an admin user account and can sign in successfully with it. 

When I access the Url Manager tab, it shows:
_Urls Manager
Create, Edit, Achives your URLs
Loading..._

The Click Statistics tab shows: 
_Oops!
404 Not Found
Sorry, an error has occured, Requested page not found!_

**Comment by Ismael-Castillo** [comment]
I had same issue until I added the connection to the Azure storage table found in Configuration section [here](https://github.com/FBoucher/TinyBlazorAdmin/blob/main/deployment.md).

**Reply by fboucher** [reply]
Documentation moved into the wiki the new link is here: https://github.com/FBoucher/TinyBlazorAdmin/wiki/Deployment#configuration
